File: python/programmers/level 0/인덱스 바꾸기.py
def solution(my_string, num1, num2):
    my_string = list(my_string)
    first = my_string[num1]
    second = my_string[num2]
    my_string.pop(num1)
    my_string.insert(num1, second)
    my_string.pop(num2)
    my_string.insert(num2, first)
    answer = ''.join(my_string)
    return answer

File: java/src/main/java/ps_traning/programmers/level_0/인덱스_바꾸기.java
package ps_traning.programmers.level_0;

public class 인덱스_바꾸기 {
    public String solution(String my_string, int num1, int num2) {
        String answer = "";
        char first = my_string.charAt(num1);
        char second = my_string.charAt(num2);
        char[] temp = my_string.toCharArray();
        temp[num1] = second;
        temp[num2] = first;
        answer = String.valueOf(temp);
        return answer;
    }
}
